crates/wasmtime/Cargo.toml

```diff
@@ -116,8 +120,11 @@ posix-signals-on-macos = ["wasmtime-runtime/posix-signals-on-macos"]
 component-model = [
   "wasmtime-environ/component-model",
   "wasmtime-cranelift?/component-model",
+  "wasmtime-winch?/component-model",
   "wasmtime-runtime/component-model",
   "dep:wasmtime-component-macro",
   "dep:wasmtime-component-util",
   "dep:encoding_rs",
 ]
+
+winch = ["dep:wasmtime-winch"]
```

Comment:
Could you put this `winch` feature declaration next to the `cranelift` feature above? A brief comment resembling the one for the `cranelift` feature would be nice too.